File: week-02/day-2/draw-triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

number = 5 #int(input("enter a number: "))

x = 1
while (x <= number):
    print("*" * x)
    x = x + 1

File: week-07/day-2/draw-triangle.js
'use strict';

// var lineCount = 4;

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

function drawTriange (lineCount) {
    var star = '*';
    for (var line = 1; line < lineCount + 1; line++) {
        console.log(star);
        star += '*';
    }
}

drawTriange (5);
